Cant Build on MacOs 13.1 ARM

title says it all

Hi,
I ran into the same issue, and as this issue was already open, I thought I'd comment here in case it's still being worked on.

Here's the error I'm getting on Apple Silicon:

```
~> pip install oead
Collecting oead
  Using cached oead-1.2.6.post1.tar.gz (3.9 MB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: oead
  Building wheel for oead (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [67 lines of output]
      /Users/neeby/.repositories/clones/testing/BCML/venv/lib/python3.9/site-packages/setuptools/dist.py:529: UserWarning: Normalizing 'v1.2.6-1' to '1.2.6.post1'
        warnings.warn(tmpl.format(**locals()))
      running bdist_wheel
      running build
      running build_ext
      -- The CXX compiler identification is AppleClang 14.0.3.14030022
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- pybind11 v2.10.1
      -- Found PythonInterp: /Users/neeby/.repositories/clones/testing/BCML/venv/bin/python3.9 (found suitable version "3.9.16", minimum required is "3.6")
      -- Found PythonLibs: /opt/homebrew/opt/python@3.9/Frameworks/Python.framework/Versions/3.9/lib/libpython3.9.dylib
      -- Performing Test HAS_FLTO
      -- Performing Test HAS_FLTO - Success
      -- Performing Test HAS_FLTO_THIN
      -- Performing Test HAS_FLTO_THIN - Success
      CMake Error at CMakeLists.txt:46 (add_subdirectory):
        The source directory

          /private/var/folders/2x/yc3gqnh578n2xf585cps14h0bqshdy/T/pip-install-ajfd30z1/oead_12fe5fd52fa74af8b6c87bcde23d07b0

        does not contain a CMakeLists.txt file.
```

I'm able to install just fine by cloning this repository and then running `pip install .`. It's only the PyPI distribution that seems to be missing some required CMake files.

I also downloaded the tarball from PyPI to see if there were any differences in the CMakeLists.txt files:

```
~> tar --list -f ~/Downloads/oead-1.2.6.post1.tar.gz | grep 'CMakeLists.txt$' | sort | sed 's/oead-1.2.6.post1/./g' > pypi_cmake.txt
~> find . -type f -name 'CMakeLists.txt' | sort > repo_cmake.txt
~> diff *cmake.txt
0a1
> ./CMakeLists.txt
```
It seems the root `CMakeLists.txt` is missing?

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,3 +1,6 @@
 include versioneer.py
 include readme.rst
+include CMakeLists.txt
+include CMakeRC.cmake
+recursive-include src *
 recursive-include lib *.*

--- a/setup.py
+++ b/setup.py
@@ -81,5 +81,6 @@ setuptools.setup(
     ],
     python_requires=">=3.6",
     ext_modules=[CMakeExtension(name="oead", sourcedir="py")],
+    data_files=[('data', [str(p) for p in Path('data').glob('**/*')])],
     zip_safe=False,
 )

